server/src/infra/repositories/album.repository.ts

```diff
@@ -36,6 +38,7 @@ export class AlbumRepository implements IAlbumRepository {
     return this.repository.findOne({ where: { id }, relations, order });
   }
 
+  @GenerateSql({ params: [DummyValue.UUID] })
```

Comment:
I think this should be `[[DummyValue.UUID]]`, correct?

Reply:
Good catch